ui-backend/catalog-ui-oauth/src/main/java/org/codice/ddf/catalog/ui/oauth/app/OAuthApplication.java

```diff
@@ -201,8 +202,11 @@ String processCodeFlow(Response res, String code, Map<String, Object> state) {
 
     JWT idToken = oidcTokens.getIDToken();
     AccessToken accessToken = oidcTokens.getAccessToken();
-    RefreshToken refreshToken = oidcTokens.getRefreshToken();
+    RefreshToken refreshToken = null;
 
+    if (!userId.startsWith(GUEST)) {
```

Comment:
Edge case, but maybe `userId.equals(GUEST)` would be better: https://en.wikipedia.org/wiki/Guest_(surname)

Reply:
The userId is usually `Guest@<IP> that's why I was using `startsWith`

Reply:
If it's always in that format, maybe including the `@` in the `startWith` check could catch that edge case

Reply:
❗️ I tested this and was able to use `SubjectUtils.isGuest(...)`. That would be the best option